packages/reactivity/src/effect.ts

```diff
@@ -278,6 +300,7 @@ export function trackEffect(
 }
 
 const queueEffectSchedulers: (() => void)[] = []
+const pausedQueueEffects = new WeakSet<ReactiveEffect>()
```

Comment:
Should we consider releasing this memory when it is destroyed?

Reply:
`WeakSet` does not prevent garbage collection.

Reply:
What about queueEffectSchedulers?

Reply:
The `pausedQueueEffects` is used to store ReactiveEffects that have been triggered during a pause. When a ReactiveEffect resumes, it immediately executes the ReactiveEffects saved in `pausedQueueEffects`. Once a ReactiveEffect is destroyed, `pausedQueueEffects` releases its reference to the ReactiveEffect.

Reply:
if ReactiveEffect is destroyed, I think we should let `queueEffectSchedulers.length=0` be able to recycle as soon as possible

Reply:
I didn't quite understand the issue you described. When an object is saved in a WeakSet, the reference to it will be automatically released when the object is destroyed. This does not lead to memory leaks. Additionally, it's important to note that WeakSet does not have a `length` property.


https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet